sign_oca/models/sign_oca_request.py

```diff
@@ -362,14 +362,13 @@ def _compute_res_id(self):
         for item in self.filtered(lambda x: x.request_id.record_ref):
             item.res_id = item.request_id.record_ref.id
 
-    @api.depends("signed_on", "partner_id", "partner_id.commercial_partner_id")
+    @api.depends("signed_on", "partner_id", "partner_id")
```

Comment:
Remove duplicated depends

```suggestion
    @api.depends("signed_on", "partner_id")
```

Reply:
Thanks for mentioning that! Fixed)